wikihow2zim/entrypoint.py

```diff
@@ -47,7 +47,7 @@ def main():
     parser.add_argument(
         "--output",
         help="Output folder for ZIM file",
-        default="/output",
+        default="output",
```

Comment:
We use `/output` as default on all scrapers because we distribute mainly through docker and it's easier to have a predictable path to advertise mounting point. I reckon it's a bit annoying when using locally but let's keep it as `/output` until we decide of a better default and apply that to all scrapers.